### DISCUSSION TITLE
simulate function gives strange results

### DISCUSSION THREAD

**Discussion by IsaDio** [pastas/pastas#624] [Q&A] [answered]
Hi,

I have been using the simulate function to run predictions (varying climate and pumping) with sets of calibrated parameters for different projects and always get great results. But I have run into some puzzling results today and not sure how I can fix this. Might be an issue with the calibration?

I first calibrated the model (Hantush with 44 bores and Flex recharge model) to the complete time series and achieved a good calibration.
![image](https://github.com/pastas/pastas/assets/91039837/b28f28db-48d4-4ece-b201-9a9e63452967)

I then decided to remove the post-2022 data and use that latest period as validation. Doing so I also get a good calibration but slightly different and maybe not as good (as expected?).
![image](https://github.com/pastas/pastas/assets/91039837/47d63cf4-f986-461c-8957-2f62bfb786fd)

I then used the simulate function to validate the second calibration with the post-2022 pumping and climate data. The simulated levels are consistent with the calibration, and look reasonable until mid-2022 when they start to diverge and plunge into - elevations. Pumping and climate should be ok as I didn't change the original input. Not sure what is happening?
![image](https://github.com/pastas/pastas/assets/91039837/c9c00754-b20e-4c41-9240-2ec9f0d156ce)

Thank you for your help!!

**Comment by IsaDio** [comment, marked as answer]
I know the question was a bit vague... but I found the answer recently. I produced this example quickly and forgot to check if new bores were being turned on during the prediction period. For new bores I usually manually set the parameters (based on nearby bores) but it looks like PASTAS assigned some default or really wrong numbers for these two bores which generated this result (huge contribution). All good, my fault :)

**Reply by dbrakenhoff** [reply]
Hi @IsaDio, 

Sorry we never got around to checking out your question, but good to see you managed to find an answer and reported back here! We appreciate that :). Your method of setting the initial parameters based on the results for nearby bores sounds interesting! Did you only use those as initial parameters and let the model solve after that or did you use those as final parameters?

Anyway, if you have any further questions or interesting results, feel free to share! 

Cheers,

Davíd

**Reply by IsaDio** [reply]
Hi @dbrakenhoff,

I am investigating to see if we could use PASTAS to do short-term (1-2 years) predictions of groundwater levels at mine  sites with active pit dewatering. I am trying different model setups with the Hantush and HantushWellModel, and getting some interesting results so far (I think). How to configure the PASTAS model for a site greatly depends on the data available and the size of the well field. I have tested sites with 1-2 bores and up to +40 bores. 
I am trying to assess how well the models perform when new bores are added during predictions (no observation available yet). This is taken care off using the HantushWellModel by providing a distance. For the Hantush model I have been trying different options, one being adding the well for predictions and assigning the parameters from existing bores (closest one?). The results are often quite reasonable.
